ui-kit/src/main/kotlin/org/p2p/uikit/components/finance_block/MainCellStyle.kt

```diff
@@ -1,5 +1,5 @@
 package org.p2p.uikit.components.finance_block
 
-enum class FinanceBlockStyle {
+enum class MainCellStyle {
     FINANCE_BLOCK, BASE_CELL
```

Comment:
can we add some figma links to ui kit here to distinct between base_cell and finance block?